prow/integ-galley-k8s-tests.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
Can you reuse lib.sh instead of copy code?

Reply:
I don't think any of the code here is in lib.sh?